### DISCUSSION TITLE
3(or more) different projects?

### DISCUSSION THREAD

**Discussion by amccool** [serilog-contrib/serilog-sinks-grafana-loki#105] [Q&A] [answered]
why is there so much fragmentation in serilog for loki support?

https://github.com/JosephWoodward/Serilog-Sinks-Loki
https://www.nuget.org/packages/Serilog.Sinks.Loki/

-or-
https://github.com/serilog-contrib/serilog-sinks-grafana-loki
https://www.nuget.org/packages/Serilog.Sinks.Grafana.Loki

-or-
https://github.com/mishamyte/serilog-sinks-grafana-loki    (listed in https://www.nuget.org/packages/Serilog.Sinks.Grafana.Loki)

then a large mismash of nugets
https://www.nuget.org/packages?q=serilog+loki

**Comment by mishamyte** [comment, marked as answer]
Hi @amccool,

You could see the comparison [here](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Comparison-with-another-Loki-sinks) (it will be updated after v8 release)
